friend-pane/utils/socket-utils.js

```diff
@@ -0,0 +1,119 @@
+import chalk from "chalk";
+import fs from "fs";
+
+import users from "../users.json" assert { type: "json" };
+
+export const inviteUser = (userEmail, onSuccess, onFail) => {
```

Comment:
The user methods could have been put in a `UsersService` instead of the `SocketUtils`, but it works either way and is only relevant if the project were bigger